subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/ContinuousBuildActionExecuter.java

```diff
@@ -77,6 +78,9 @@ public ContinuousBuildActionExecuter(
 
     @Override
     public BuildActionResult execute(BuildAction action, BuildRequestContext requestContext, final BuildActionParameters actionParameters, ServiceRegistry buildSessionScopeServices) {
+        // Ensure that file system watching is enabled
+        // TODO: Use the handler later on to do things with continuous build
+        buildSessionScopeServices.get(FileSystemWatchingHandler.class);
```

Comment:
Why is this needed?

Reply:
See https://gradle.slack.com/archives/CDYR952SH/p1594659521457500

I'll add a clarifying comment.

Reply:
Added.